docs/basic_training/intro.es.md

```diff
@@ -0,0 +1,274 @@
+
+Los procesos se ejecutan de forma independiente y están aislados entre sí, es decir, no comparten un estado común (de escritura). La única forma en que pueden comunicarse es a través de colas FIFO (primero en entrar, primero en salir) asincrónicas, llamadas "canales".
+
+Cualquier `proceso` puede definir uno o más `canales` como `entrada` y `salida`. La interacción entre estos procesos y, en última instancia, el propio flujo de ejecución del flujo, se define implícitamente mediante estas declaraciones de "entrada" y "salida".
+
+<figure class="excalidraw">
+--8<-- "docs/basic_training/img/channel-process.excalidraw.svg"
+</figure>
+
+### Abstracción de ejecución
+
+Mientras que un 'proceso' define _qué_ comando o 'script' debe ejecutarse, el ejecutor determina _cómo_ se ejecuta ese 'script' en la plataforma de destino.
+
+Si no se especifica lo contrario, los procesos se ejecutan en la computadora local. El ejecutor local es muy útil para desarrollar flujos y con fines de prueba; sin embargo, para los flujos computacionales del mundo real, a menudo se requiere una plataforma de alto rendimiento (HPC) o la nube.
+
+En otras palabras, Nextflow proporciona una abstracción entre la lógica funcional del flujo y el sistema de ejecución (o runtime) subyacente. Por lo tanto, es posible escribir un flujo que se ejecute sin problemas en su computadora, un clúster o la nube, sin ser modificada. Simplemente defina la plataforma de ejecución de destino en el archivo de configuración.
+
+<figure markdown>
+
+![Execution abstraction](img/execution_abstraction.png)
+
+</figure>
+
+### Lenguaje de escritura
+
+Nextflow implementa un DSL declarativo que simplifica la escritura de flujos de trabajo de análisis de datos complejos como una extensión de un lenguaje de programación de propósito general.
+
+Este enfoque hace que Nextflow sea flexible: brinda los beneficios de un DSL conciso para el manejo de casos de uso frecuente con facilidad **y** da la flexibilidad y el poder de un lenguaje de programación de propósito general para manejar casos más complejos en el mismo entorno. Esto sería difícil de implementar utilizando un enfoque puramente declarativo.
+
+En términos prácticos, las secuencias de comandos de Nextflow son una extensión del [lenguaje de programación Groovy] (https://groovy-lang.org/) que, a su vez, es un superconjunto del lenguaje de programación Java. Groovy puede considerarse como "Python para Java", ya que simplifica la escritura de código y es más accesible.
+
+## Tu primer script
+
+Aquí ejecutará su primer script de Nextflow (`hello.nf`), que revisaremos línea por línea.
+
+En este ejemplo sencillo, el script toma una cadena de entrada (un parámetro llamado `params.saludo`) y la divide en partes de seis caracteres en el primer proceso. El segundo proceso convierte los caracteres a mayúsculas. El resultado finalmente se muestra en la pantalla.
```

Comment:
In my opinion, I would use "el script toma un **string** como entrada". I get a bit lost with 'cadena de entrada'. Portuguese translation also kept the anglicism _string_.